exporter/awscloudwatchlogsexporter/README.md

```diff
@@ -44,21 +42,23 @@ exporters:
     log_stream_name: "testing-integrations-stream"
 ```
 
-All configuration options:
+Example configuration for EMF logs:
 
 ```yaml
 exporters:
   awscloudwatchlogs:
-    log_group_name: "testing-logs"
-    log_stream_name: "testing-integrations-stream"
+    log_group_name: "testing-logs-emf"
+    log_stream_name: "testing-integrations-stream-emf"
+    raw_log: true
     region: "us-east-1"
-    role_arn: "arn:aws:iam::123456789:role/monitoring-application-logs"
     endpoint: "logs.us-east-1.amazonaws.com"
     log_retention: 365
     tags: { 'sampleKey': 'sampleValue'}
-    sending_queue:
-      queue_size: 50
-    retry_on_failure:
-      enabled: true
-      initial_interval: 10ms
 ```
+
+## Additional Notes 
+
+- If the log group and/or log stream are specified in an EMF log, that EMF log will be exported to that log group and/or log stream (i.e. ignores the log group and log stream defined in the configuration)
+  - The log group and log stream will also be created automatically if they do not already exist
+- [ARNs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) are currently not supported with the CloudWatch Logs Exporter
```

Comment:
Actually I confused what isn't used and what is used regarding ARNs, thanks for this comment. What isn't used is resource_arns (Amazon Resource Name (ARN) of the AWS resource running the collector), and from my understanding resource_arns is only used with [xray](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsxrayexporter) as of now. I'll change the note to clarify this and also re-add the role_arns configuration as I removed it before.

Reply:
The reason why I initially thought that role_arns wasn't supported was because of this comment in [config.go](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awscloudwatchlogsexporter/config.go)